pkg/meta/meta.go

```diff
@@ -7,10 +7,12 @@ import (
 	"encoding/xml"
 	"fmt"
 	"strings"
+	"time"
 )
 
 type LibvirtProviderMetadata struct {
-	IRIMmachineLabels string `xml:"irimachinelabels"`
+	IRIMmachineLabels string    `xml:"irimachinelabels"`
```

Comment:
it looks as mixing of 2 key format without _ and with _. I prefer unified naming convetion